tests/modules/sightings/resources/test_sighting_elasticsearch.py

```diff
@@ -30,3 +33,25 @@ def no_test_sighting_elasticsearch_mappings(flask_app_client, researcher_1):
         expected_status_code=200,
         response_200=EXPECTED_KEYS,
     )
+
+
+def test_search(flask_app_client, researcher_1, request, test_root):
+    from app.modules.sightings.models import Sighting
```

Comment:
Needs to check for Codex being enabled, this fails for MWS test.

Reply:
Ah right, of course, let me fix it